.github/workflows/otbr.yml

```diff
@@ -152,6 +164,7 @@ jobs:
       PYTHONUNBUFFERED: 1
       VERBOSE: 1
       BORDER_ROUTING: 1
+      NAT64: ${{ matrix.nat64 }}
```

Comment:
We are disabling the nat64 by default since the native nat64 support is not merged, how about adding a comment for this for re-enable it later.

Reply:
Currently NAT64 is disabled by default in all tests (see [here](https://github.com/openthread/openthread/blob/main/script/test#L327)), while this PR is enabling it in nat64 related tests.